schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
+ CREATE DATABASE  vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+   id INT,
```

Comment:
- [ ] [optional] @rushemaa Great work creating a new table, you could also have made id not null and made it to autoincrement as new data is added to the table by using this code , also adding it as primary key
```   
 id int NOT NULL IDENTITY(1,1)
 PRIMARY KEY (id)

```